confd/tests/mock_data/calicoctl/explicit_peering/route_reflector/input.yaml

```diff
@@ -78,6 +78,7 @@ metadata:
 spec:
   bgp:
     ipv4Address: 10.192.0.3/16
+    ipv6Address: fe0a::3/96
```

Comment:
Added IPv6 addresses to the old test so that it covered the case with a peering based on selector, which _should_ generate peerings for IPv4 and IPv6.